services/horizon/internal/db2/schema/migrations/38_add_constraints.sql

```diff
@@ -0,0 +1,115 @@
+-- +migrate Up
+
+ALTER TABLE history_transactions ADD CONSTRAINT valid_ledger_sequence CHECK (ledger_sequence >= 0) NOT VALID;
```

Comment:
There are many other fields that should be `>` instead of `>=`: fees, last_modified_ledger. Can you take another look at this?
```suggestion
ALTER TABLE history_transactions ADD CONSTRAINT valid_ledger_sequence CHECK (ledger_sequence > 0) NOT VALID;
```